packages/v4/patternfly-docs/content/design-guidelines/styles/colors/colors.md

```diff
@@ -87,72 +87,69 @@ import './colors.css';
     </ColorSwatch>
-    <ColorSwatch color="--pf-global--success-color--100" caption="alert icon background">
-      This color is most commonly used as an indication of success for components, such as alerts and forms. It is used as the background color for the success alert component as well as the background and icon color for the success inline alert component.
+    <ColorSwatch color="--pf-global--success-color--100" caption="alert icon">
+      This color is used as the icon color for the success alert component.
+    </ColorSwatch>
+    <ColorSwatch color="--pf-global--palette--green-50" caption="alert background">
+      This color is used as the background color for the success inline alert component.
     </ColorSwatch>
   </GridItem>
   <GridItem xl={6} lg={12}>
-    <h3>Warning</h3>
-    <ColorSwatch color="--pf-global--warning-color--200" caption="alert title and icon">
-      This color is used as the icon and title color for the warning alert component.
+    <h3>Info</h3>
+    <ColorSwatch color="--pf-global--info-color--200" caption="alert title">
+      This color is used as the title color for the info alert component.
+    </ColorSwatch>
+    <ColorSwatch color="--pf-global--info-color--100" caption="alert icon">
+      This color is used as the icon color for the default alert component.
     </ColorSwatch>
-    <ColorSwatch color="--pf-global--warning-color--100" caption="alert icon background">
-      This color is used as the icon background color for the warning alert component as well as the background and icon color for the warning inline alert component.
+    <ColorSwatch color="--pf-global--palette--blue-50" caption="alert background">
+      This color is used as the background color for the info inline alert component.
     </ColorSwatch>
   </GridItem>
   <GridItem xl={6} lg={12}>
-    <h3>Disabled</h3>
-    <ColorSwatch color="--pf-global--disabled-color--100">
-      This color is used as the disabled text color when text is indicated as disabled.
+    <h3>Warning</h3>
+    <ColorSwatch color="--pf-global--warning-color--200" caption="alert title">
+      This color is used as the title color for the warning alert component.
     </ColorSwatch>
-    <ColorSwatch color="--pf-global--disabled-color--200">
-      This color is most commonly used as a disabled background color for many components, such as buttons and the app launcher.
+    <ColorSwatch color="--pf-global--warning-color--100" caption="alert icon">
+      This color is used as the icon color for the warning alert component.
     </ColorSwatch>
-    <ColorSwatch color="--pf-global--disabled-color--300">
-      This color is most commonly used as an alternate disabled background color for many components, such as dropdowns and options menus.
+    <ColorSwatch color="--pf-global--palette--gold-50" caption="alert background">
+      This color is used as the background color for the warning inline alert component.
     </ColorSwatch>
```

Comment:
We don't have the disabled colors in this section any more. Should we? If not, should it be taken out of the accompanying graphic?

Reply:
Nope - that was a total mistake! I added it back in my last update.